serverlessworkflow/antora.yml

```diff
@@ -125,6 +127,15 @@ asciidoc:
     golang_install_url: https://go.dev/doc/install
     serverless_logic_web_tools_url: https://start.kubesmarts.org/
     swf_executor_core_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-core
+    swf_fluent_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-fluent
+    swf_executor_rest_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-rest
+    swf_executor_python_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-python
+    swf_executor_grpc_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-grpc
+    swf_executor_events_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-kafka
+    swf_executor_service_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-executor-service
+    swf_executor_openapi_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-serverless-workflow-openapi-parser
+    rocksdb_addon_maven_repo_url: https://mvnrepository.com/artifact/org.kie.kogito/kogito-addons-persistence-rocksdb
```

Comment:
These URLs will change in the product/midstream? cc @rgdoliveira @domhanak otherwise, I don't see a reason to keep them here instead on their on adoc file.

Reply:
The URLS will change, both for midstream and when we rename them to sonata flow (if we implement issue https://github.com/apache/incubator-kie-kogito-runtimes/issues/3447)